README.md

```diff
@@ -401,6 +402,31 @@ apk del grep sed curl fzf git aria2 ffmpeg ncurses
 - ffmpeg - m3u8 Downloader (fallback)
 - fzf - User interface
 
+### Ani-Skip (optional)
```

Comment:
it should have a bullet point for good measure similarly to ffmpeg and the others

Reply:
I added it to the list.
Can I leave the ani-skip section as it is now?